sr_gui_joint_slider/src/sr_gui_joint_slider/joint_slider.py

```diff
@@ -355,10 +334,8 @@ def _create_joints(self, controllers):
                             if self._widget.joint_name_filter_edit.text() not in j_name:
                                 continue
 
-                            min, max, vel = self._get_joint_min_max_vel_special(
-                                j_name)
-                            joint = Joint(
-                                j_name, min, max, vel, joint_controller)
+                            min_value, max_value, vel = self._get_joint_min_max_vel_special(j_name)
```

Comment:
If we change the names maybe it makes more sense to call these variables `max_position`, `min_position`, `max_velocity` (velocity limit is the same in both directions).